shardingsphere-features/shardingsphere-readwrite-splitting/shardingsphere-readwrite-splitting-core/src/main/java/org/apache/shardingsphere/readwritesplitting/algorithm/loadbalance/RandomReplicaLoadBalanceAlgorithm.java

```diff
@@ -39,6 +40,9 @@ public void init(final Properties props) {
     
     @Override
     public String getDataSource(final String name, final String writeDataSourceName, final List<String> readDataSourceNames) {
+        if (TransactionHolder.isTransaction()) {
```

Comment:
@jingshanglu Since our load balance algorithm also includes the logic of the primary datasource, do we need to rename ReplicaLoadBalanceAlgorithm to ReadWriteSplittingLoadBalanceAlgorithm?

Reply:
I think `replica` can express that.